doc/deployment/deployment.md

```diff
@@ -46,51 +35,44 @@ If I modified native code, I need to hard deploy:
 
 #### Staging
 
-We do it once a week at the end of an iteration.
+![img](./MES.png)
 
 When you want to deploy the current version of master in staging, you can run the following command:
 
 - `yarn trigger:staging:deploy`
-  The first time you will be asked about username and password.
-  Username = your github username
-  Password = your token (you created here https://github.com/settings/tokens with at least repo scope)
 
 This will create a tag `vX.X.X` and push it.
 
-And this will create 2 pull request from MES-X.X.X to `staging` branch and `master` branch.
-
-- Merge the 2 PR when tests are green.
-
-CircleCI will detect the merge on `staging` branch and launch the lanes `deploy-ios-staging` & `deploy-android-staging` (see `.circleci/config.yml` file)
+CircleCI will detect the tag `vX.X.X` and launch the lanes `deploy-ios-staging-hard` & `deploy-android-staging-hard` (see `.circleci/config.yml` file)
 
 ## Production: Deploy to App Store / Google Play Store
 
 ### Deploy hard
 
-To deploy to production:
-
-- `git checkout staging`: because we want to deploy what have been tested by PO on staging env (last tag vX.X.X)
+![img](./MEP.png)
 
-- `yarn trigger:production:deploy`
-  And this will create a pull request from `staging` to `production` branch.
-  CircleCI will detect the tag and launch the lane `deploy-android-production-hard` & `deploy-ios-production-hard` (see `.circleci/config.yml` file)
+- Know which version (and then tag) you want to deploy
 
-### Deploy CodePush
+- `yarn trigger:production:deploy <tag>`
 
-#### Stratégie
+This will create a tag `prod-hard-deploy`
+CircleCI will detect the tag and launch the lane `deploy-android-production-hard` & `deploy-ios-production-hard` (see `.circleci/config.yml` file)
 
-![img](./codepush-strategy.png)
+## Hotfix
 
-#### How to
+#### Strategy
 
-/!\ DO NOT DEPLOY CODEPUSH IOS & ANDROID SIMULTANEOUSLY
+![img](./HOTFIX.png)
 
-- `git checkout hotfix/vX.X.X` (X.X.X version currently in production)
-- cherry-pick all fix commits
-- git push
-- wait for the CI to be green
-- deploy:
-
-  1. Android: `yarn trigger:production:codepush:android`
+#### How to
 
-  2. When android finished only, IOS: `yarn trigger:production:codepush:ios`
+- check tags of the version: `vX.X.X-Y`
+- checkout on the biggest Y (if no tag with Y, checkout on `vX.X.X`)
+- code the fix
```

Comment:
cherry-pick the commit that you need

Reply:
en fait ça dépend de si t'as codé le fix avant sur master ou non ? 
Et je suis partie de l'idée que si t'as besoin de faire un hotfix c'est que t'as un fix urgent à faire, donc tu commences par protéger la prod. 
Tu codes le fix, tu testes sur staging (env iso prod) puis t'envoies en prod puis tu rappatries le fix sur master